modules/networkfirewall-dashboard/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
This has been reviewed. And necessary changes have been made to pass github actions CI and local terraform apply.